types/google.accounts/index.d.ts

```diff
@@ -38,6 +38,12 @@ declare namespace google.accounts {
          */
         function revoke(accessToken: string, done: () => void): void;
 
+        interface ClientConfigError extends Error {
+            message: string;
+            stack?: string;
+            type: 'unknown' | 'popup_closed' | 'popup_failed_to_open';
+        }
```

Comment:
The `type` field is clearly documented at the supplied link; did you find any other documentation indicating this object extends `Error`? I haven't actually encountered it myself and I'm content to trust you're accurate here (worst case it gets fixed in a future PR), but I would like to see docs if they exist.